realsense_camera/README.md

```diff
@@ -4,24 +4,47 @@
 | Version          | Best Known (indigo)  | Best Known (kinetic) |
 |:---------------- |:---------------------|:---------------------|
 | Operating System | Ubuntu 14.04.4 LTS   | Ubuntu 16.04 LTS     |
-| Kernel           | 4.4.0-040400-generic | 4.4.0-22-generic     |
+| Kernel           | 4.4.0-34-generic     | 4.4.0-34-generic     |
 | Backend          | video4linux          | video4linux          |
 | librealsense     | 0.9.2                | 0.9.2                |
 | R200 Firmware    | 1.0.72.06            | 1.0.72.06            |
 | F200 Firmware    | 2.60.0.0             | 2.60.0.0             |
 | SR300 Firmware   | 3.10.10.0            | 3.10.10.0            |
 
 ###Installation:
-#####Getting the camera to work on Linux
+####Getting the camera to work on Linux
 
-* Clone the source from the librealsense git repository https://github.com/IntelRealSense/librealsense.git and follow the "Installation Guide" for installing the library.
-* Make sure that the software stack is installed properly and that the camera is working. This can be checked by connecting the camera to a USB3 port and running the "cpp-capture" sample program in the "librealsense/bin" folder.
-If this does not work, you should first fix this issue before continuing with the ROS integration.
-* Make sure "/usr/local/lib" is set in your "LD_LIBRARY_PATH".
+* Install ROS (see [wiki.ros.org/ROS/Installation](http://wiki.ros.org/ROS/Installation) for instructions).
 
-#####Building the package
+* Install the realsense_sense camera package
 
-* Install ROS and create a local catkin workspace (see [wiki.ros.org](http://wiki.ros.org/) for instructions).
+  `sudo apt-get install ros-<*distro*>-realsense_camera`
+
+  This will also install the required `ros-<*distro*>-librealsense` library on your system.
+
+**NOTE:** Replace `<*distro*>` with `kinetic` or `indigo`.
+
+##### Librealsense REQUIRED Manual Installation Steps
+librealsense 0.9.2 and earlier require a few extra, manual steps to enable the use of the library.
+
+* Clone the librealsense GitHub repository:
+
+```
+  git clone https://github.com/IntelRealSense/librealsense.git
+  cd librealsense
+  git checkout -b b0.9.2 v0.9.2
+```
+
+* Follow the instructions for [Video4Linux
+  backend](https://github.com/IntelRealSense/librealsense/blob/master/doc/installation.md#video4linux-backend) **ONLY** 
+
+  **NOTE:** Only one of the three options for Step 3 should be completed
+
+**NOTE:** The next release (0.9.3?) of `ros-<*distro*>-librealsense` will no longer require these manual steps.
```

Comment:
Do we need this "(0.9.3?)" ? We could just say "The next release of `ros-<*distro*>-librealsense` will no longer require these manual steps."